src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -282,7 +280,16 @@ function main() {
   ) {
     redirectIfNecessary(window);
     removeCurrentHeaderFooter();
-    activateInjectedAssets();
+
+    if (
+      document.readyState === 'complete' ||
+      document.readyState === 'loaded' ||
+      document.readyState === 'interactive'
+    ) {
+      activateInjectedAssets();
+    } else {
+      document.addEventListener('DOMContentLoaded', activateInjectedAssets);
```

Comment:
- add readystate check before listening for `DOMContentLoaded` 

`DOMContentLoaded` doesn't new listeners if it's already been called.